llvm/test/MC/Disassembler/AMDGPU/gfx12_dasm_sopp.txt

```diff
@@ -1,4 +1,5 @@
-# RUN: llvm-mc -triple=amdgcn -mcpu=gfx1200 -disassemble -show-encoding < %s | FileCheck -strict-whitespace -check-prefixes=GFX12 %s
+# RUN: llvm-mc -triple=amdgcn -mcpu=gfx1200 -disassemble -show-encoding < %s | FileCheck -strict-whitespace -check-prefixes=GFX12,GFX1200 %s
+# RUN: llvm-mc -triple=amdgcn -mcpu=gfx1250 -disassemble -show-encoding < %s | FileCheck -strict-whitespace -check-prefixes=GFX12 %s
```

Comment:
How does this work without a `not`? Doesn't the disassembler give an error on the unrecognized gfx1200-only instructions?

Reply:
Like this:
```
llvm-mc -arch=amdgcn -mcpu=gfx1250 -mdis <<< 0x34,0x12,0xc4,0xbf
<stdin>:1:1: warning: invalid instruction encoding
0x34,0x12,0xc4,0xbf
```
But llvm-mc still returns exit code 0.